### PR TITLE
Ability to add custom stylesheet

### PR DESCRIPTION
**Pull request by kfuglsang**
Fix for #44

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -31,6 +31,14 @@ Target "Clean" (fun _ ->
     CleanDirs [outDir]
 )
 
+let copyStylesheet() =
+    try
+        let cssFile = (slidesDir @@ "custom.css")
+        if (fileExists cssFile) then
```
Comment (forki) [inline build.fsx]: please add a default file instead. that will make it easier for people to understand what to do

**Comment by forki** [comment]
you need to run `.paket/paket.exe update` to get the latest FsReveal package. then don't forget to commit paket.lock

**Comment by kfuglsang** [comment]
Done. Good idea.

**Comment by forki** [comment]
Very cool. Case closed

**Comment by tpetricek** [comment]
This was broken on Mac. I did a direct commit that fixes the path: https://github.com/fsprojects/FsReveal/commit/0629b1954801b31a7f7db4ae9a53f8a64d981fdf

**Pull request merged**